app/components/appcomponents/ModalDigiCards.js

```
@@ -0,0 +1,95 @@
+                        </div>
+                        <div className="flex w-[100%] mobile:w-[344px] z-50 mobile:px-[2px] px-7 pb-11 mobile:mt-11 mt-12  items-center justify-center">
+                            <div className="mobile:w-[314px] w-[910px] bg-[#E1E6EC]  rounded-2xl border-[#6D778E] border pt-12 mobile:px-6 px-8 pb-7 flex flex-col">
+                                <h1 className="text-[#1E2125] text-2xl mobile:text-xl font-medium mb-2.5 text-center">
+                                    Moje darilne kartice
+                                </h1>
+
+                                {data && data?.length ? (
+                                    <div className="flex justify-center mt-5">
+                                        {data?.map((item) => {
+                                            return (
+                                                <div className="w-[23%]">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)